src/payment/strategies/paypal-commerce/paypal-commerce-request-sender.ts

```diff
@@ -57,4 +57,11 @@ export default class PaypalCommerceRequestSender {
 
         return res.body;
     }
+
+    async deleteCart(cartId: string) {
```

Comment:
(optional) It might be better to create `CartRequestSender` class and move this method there